### PR TITLE
Add headers to the webhook response

### PR DESCRIPTION
**Pull request by gonzalojaubert**
Add headers to the webhook response. This is a **breaking change** as the webhook handler now returns an object. It should be deployed as 0.2.0 version

Also:
* Update to latest booster version

### CODE REVIEW

File: packages/rocket-webhook-core/src/webhook-dispatcher.ts [hunk 1]
```diff
@@ -20,26 +21,29 @@ export async function dispatch(
     const handlerClass = webhookParamsEvent.handlerClass
     const webhookEvent = toWebhookEvent(webhookParamsEvent, request)
     const register = new Register(requestId, {})
-    const result = await handlerClass.handle(webhookEvent, register)
+    const result: WebhookHandlerReturnType | void = await handlerClass.handle(webhookEvent, register)
     await RegisterHandler.handle(config, register)
+    if (result) {
+      return config.provider.api.requestSucceeded(result.body, result.headers)
+    }
     return config.provider.api.requestSucceeded(result)
```
Comment (alvaroloes) [inline packages/rocket-webhook-core/src/webhook-dispatcher.ts]: I guess in this case we are calling `requestSucceeded` always with the first parameter set to `undefined`, correct? 

If so, could you please change this to `config.provider.api.requestSucceeded()` (if possible by the parameter type)?

This way is clearer we are sending an empty response (I got a bit confused)

**Review by alvaroloes — APPROVED** [review]
Minor comment. 

A clarification: As I understood, this is only a breaking change if the current Webhook handlers are returning something, but not if they return `void`. Anyway, this should be stated in the release notes

File: packages/rocket-webhook-core/src/webhook-dispatcher.ts (hunk 1 above)
Comment (gonzalojaubert) [reply packages/rocket-webhook-core/src/webhook-dispatcher.ts]: Sure, good point

**Review by alvaroloes — APPROVED** [review]

**Comment by gonzalojaubert** [comment]
Converted to draft until we update it to new Booster version and fix the package.json files

**Review by javiertoledo — APPROVED** [review]

**Pull request merged**